Assets/_BForBoss/_AI/Scripts/DummyTargetBehaviour.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Perigon.AI
+{
+    public class DummyTargetBehaviour : LifeCycleBehaviour
+    {
+        private const float DEATH_ANIMATION_DURATION = 5f;
+        private const string IS_DEAD = "IsDead";
+        private const string HIT = "Hit";
+        private readonly int DEATH_ID = Animator.StringToHash(IS_DEAD);
+        private readonly int HIT_ID = Animator.StringToHash(HIT);
+        
+        private Animator _animator;
+        
+        
+        protected override void Awake()
+        {
+            base.Awake();
+            _animator = GetComponentInChildren<Animator>();
+        }
+
+        protected override void LifeCycleFinished()
+        {
+            _animator.SetBool(DEATH_ID, true);
```

Comment:
Do you know why a shotgun shot would call this twice? And is it a huge deal or is it just placeholder.

Reply:
to be more specific, the death animation is shown twice.

Reply:
Hmm that could be an issue later on, I'll work on a fix for that